lib/avo/base_resource.rb

```diff
@@ -48,6 +48,7 @@ def current_user
     class_attribute :includes, default: []
     class_attribute :authorization_policy
     class_attribute :translation_key
+    class_attribute :base_field_translation_key
```

Comment:
Would be possible to add a default to the resource name? Ex `Avo::Resources::Project` to have as default `base_field_translation_key == "avo.field_translations.project"`

When fetching the translation can we check something like (pseudo code) `if "avo.field_translations.project.some_field".exists_translation? else ...`

This way would only be necessary to add the `base_field_translation_key` when want a different key. It would work as default if there are nested fields under a known resource scope.

Reply:
@Paul-Bob It's a good idea. So based on your suggestion, the "on/off" of this feature would be having the real translation in the yml file, correct?

If yes, we would never have the "missing translation key" error that rails or avo throws for a missing key. Is this what we want here? Am I missing something?

Reply:
Your concern makes sense and kinda invalidates my suggestion, let's keep it simple and keep the current implementation since we want to have the "missing translation key" error raised

Reply:
@Paul-Bob I am not 100% sure that this will happen, but since you have a better understanding of the code-base and you agreed, that's great. :)